app/people/customer.py

```diff
@@ -0,0 +1,10 @@
+class Customer:
+    customers = {}
+
+    def __init__(self, name: str, food: str) -> None:
+        self.name = name
+        self.food = food
+        Customer.customers[name] = self
```

Comment:
don't store customers in Customer class
```suggestion
class Customer:

    def __init__(self, name: str, food: str) -> None:
        self.name = name
        self.food = food
```